### DISCUSSION TITLE
Master reporting collection

### DISCUSSION THREAD

**Discussion by livesayjesse** [NUWCDIVNPT/stig-manager#754] [Ideas and Enhancements]
We just started using STIG-Manager and we love it! Great work btw. An item that we would like to see in the future if at all possible would be some way to view all STIGs in something like a master/main repository, even if it was read only. When leadership asked for a big picture type of report we have to gather that all from the individual collections we have. Since we have several collections that adds in a bit more work. Just a idea for future releases if its possible.

**Comment by csmig** [comment]
This is a popular request that we've also heard through other channels. We've decided to work this feature.

My current priority is to re-implement the Collection-level Status report interface to perform acceptably with large (20,000+ checklist) Collections. Some users are finding the current report interface unusable. I'll design so the new elements can be used for the new feature with reports to be made available at the Root-level.

**Reply by livesayjesse** [reply]
Outstanding! We look forward to the changes and thank you for the quick response.

**Reply by livesayjesse** [reply]
I hadn't checked in for a while but was this feature implemented? I'm sure with all the re-design it is no easy thing. I just wanted to check. I didn't see if in the latest release but the re-work of the reporting is MUCH better!

**Comment by cd-rite** [comment]
Hi @livesayjesse Glad you like the new report!!  We have some other features in the pipeline before we can prioritize the Collection roll-up report.  Some other users have started writing their own CLI clients to hit our API metrics endpoints and accomplish something similar with spreadsheet output, but they are not available on GitHub at the moment.

**Reply by livesayjesse** [reply]
No worries at all! Yeah we just made a separate collection and do a mass import from the file share. Its not 100% accurate but for when management asked for the overall numbers it gives us an idea. Thanks!
